src/card.tsx

```diff
@@ -393,3 +406,252 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
         );
     }
 );
+
+type CardAction = {
+    label: string;
+    onPress: () => void;
+    Icon: React.FC<IconProps>;
+};
+
+type CardActionsGroupProps = {
+    actions: Array<CardAction>;
+    isInverse?: boolean;
+};
+
+const CardActionsGroup = ({actions, isInverse}: CardActionsGroupProps): JSX.Element => {
+    return (
+        <Inline space={0}>
+            {actions.map(({onPress, label, Icon}, index) => (
+                <IconButton
+                    size={48}
+                    key={index}
```

Comment:
wouldn't it be better to use `useId` instead of index? as this could come form api or cache maybe it changes the order or something else and this could be broken

Reply:
I don't see how this can break

Reply:
for instance, you read from persistence cards 1,2,3
then you fetch from server, and you get cards 2,1,3 (re ordered from cms), react wont update this list

Reply:
hmmm... I don't see why

Reply:
well, someone can change the order in the CMS, or if the content come in partial responses... dont know, I think it is safer to use useId than index as a key IMO